src/main/java/frc5124/robot2020/subsystems/Shooter.java

```diff
@@ -2,160 +2,73 @@
-
-        // display PID coefficients on SmartDashboard
-        SmartDashboard.putNumber("P Gain", kP);
-        SmartDashboard.putNumber("I Gain", kI);
-        SmartDashboard.putNumber("D Gain", kD);
-        SmartDashboard.putNumber("I Zone", kIz);
-        SmartDashboard.putNumber("Feed Forward", kFF);
-        SmartDashboard.putNumber("Max Output", kMaxOutput);
-        SmartDashboard.putNumber("Min Output", kMinOutput);
-
-        SmartDashboard.putNumber("SetPoint", setPoint);
-
-    }
-
-    @Override
-    public void periodic() {
-        // Put code here to be run every loop
-        // read PID coefficients from SmartDashboard
-        double p = SmartDashboard.getNumber("P Gain", 0);
-        double i = SmartDashboard.getNumber("I Gain", 0);
-        double d = SmartDashboard.getNumber("D Gain", 0);
-        double iz = SmartDashboard.getNumber("I Zone", 0);
-        double ff = SmartDashboard.getNumber("Feed Forward", 0);
-        double max = SmartDashboard.getNumber("Max Output", 0);
-        double min = SmartDashboard.getNumber("Min Output", 0);
-
-        double setPoint = SmartDashboard.getNumber("SetPoint", 0);
-
-        // if PID coefficients on SmartDashboard have changed, write new values to controller
-        if((p != kP)) { m_pidController.setP(p); kP = p; }
-        if((i != kI)) { m_pidController.setI(i); kI = i; }
-        if((d != kD)) { m_pidController.setD(d); kD = d; }
-        if((iz != kIz)) { m_pidController.setIZone(iz); kIz = iz; }
-        if((ff != kFF)) { m_pidController.setFF(ff); kFF = ff; }
-        if((max != kMaxOutput) || (min != kMinOutput)) { 
-            m_pidController.setOutputRange(min, max); 
-            kMinOutput = min; 
-            kMaxOutput = max; 
-        }
-
-        /**
-         * PIDController objects are commanded to a set point using the 
-         * SetReference() method.
-         * 
-         * The first parameter is the value of the set point, whose units vary
-         * depending on the control type set in the second parameter.
-         * 
-         * The second parameter is the control type can be set to one of four 
-         * parameters:
-         *  com.revrobotics.ControlType.kDutyCycle
-         *  com.revrobotics.ControlType.kPosition
-         *  com.revrobotics.ControlType.kVelocity
-         *  com.revrobotics.ControlType.kVoltage
-         */
-
-        m_pidController.setReference(setPoint, ControlType.kVelocity);
+/**
+ * 
+ * WARNING 
+ * Untuned
+ * 
+ * 
+ * 
+ */
+public class Shooter implements Subsystem {
+  private CANSparkMax shootMotorFollower = new CANSparkMax(RobotMap.ShooterMap.shootFollowerCanID, MotorType.kBrushless);
+  private CANSparkMax shootMotorLeader = new CANSparkMax(RobotMap.ShooterMap.shootLeaderCanID, MotorType.kBrushless);
+  private CANPIDController shootController; 
+  private double currentVelocity;
+ 
 
-        SmartDashboard.putNumber("ProcessVariable", m_encoder.getVelocity());
+  
+  public Shooter() {
```

Comment:
why are all of the smart dashboard commands removed. This was something I noted explicitly as a benefit of the example code that we can view and modify in real time the PID tuning parameters and the velocity of the shooter.

Reply:
takes two moments to add remove. If you want me to keep them in for now I will. I usually just use them for tuning and remove later.

Reply:
I'd like them in master so we can tune the shooter easily when we get the belts to finish it